tests/5.1/order/test_parallel_for_order_reproducible.c

```diff
@@ -0,0 +1,51 @@
+//===------------ test_target_parallel_for_order_reproducible.c -------------===//
+//
+// OpenMP API Version 5.1 Nov 2020
+//
+// This test checks that the order(reproducible) clause is properly handled.
+// Leverages the previous array item to determine if the same threads executed
+// in the same order over the array. If both arrays match the test passes, 
+// if not it fails as the reproducible clause failed.
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1024
+
+int main() {
+	int errors = 0;
+	int x[N];
+	int y[N];
+
+	for (int i = 0; i < N; i++) {
+		x[i] = i;
+		y[i] = i;
+	}
+
+	OMPVV_TEST_OFFLOADING;
+
+	#pragma omp target map(tofrom: x,y)
+	{
+
+		#pragma omp parallel for order(reproducible)
```

Comment:
Fails with GCC 12 with:
```
test_parallel_for_order_reproducible.c:34:48: error: expected ‘concurrent’ before ‘reproducible’
   34 |                 #pragma omp parallel for order(reproducible)
```
Cf. https://www.openmp.org/spec-html/5.1/openmpsu47.html#x72-720002.11.3

But it works with GCC 12 after applying the change
```suggestion
		#pragma omp parallel for order(reproducible:concurrent)
```
and ...

Reply:
Good catch! I was just looking at the spec and realized I had forgotten that. Also glad to see it's working with GCC 12